scripts/hand_pose_converter.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+import json
+import os
+
+import numpy as np
+
+from angel_system.utils.matrix_conversion import (
+    convert_1d_4x4_to_2d_matrix,
+    project_3d_pos_to_2d_image,
+)
+
+
+# NOTE: These values were extracted from the projection matrix provided by the
+# Unity main camera with the Windows MR plugin (now deprecated).
+# For some unknown reason, the values provided by the Open XR plugin are
+# different and do not provide the correct results. In the future, we should
+# figure out why they are different or extract the focal length values from the
+# MediaFrameReader which provides the frames, instead of the Unity main camera.
+FOCAL_LENGTH_X = 1.6304
+FOCAL_LENGTH_Y = 2.5084
+
+PROJECTION_MATRIX = [
+    [FOCAL_LENGTH_X, 0.0, 0.0, 0.0],
+    [0.0, FOCAL_LENGTH_Y, 0.0, 0.0],
+    [0.0, 0.0, -1.0020020008087158, -0.20020020008087158],
+    [0.0, 0.0, -1.0, 0.0]
+]
```

Comment:
I assume this could also use the content now "shared" in the angel_system package.